bitnami/elasticsearch/templates/_helpers.tpl

```diff
@@ -279,33 +279,33 @@ Also, we can't use a single if because lazy evaluation is not an option
 {{/*
 Return the proper Storage Class
 */}}
-{{- define "elasticsearch.data.storageClass" -}}
+{{- define "elasticsearch.storageClass" -}}
 {{/*
 Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
 but Helm 2.9 and 2.10 does not support it, so we need to implement this if-else logic.
 */}}
```

Comment:
This function is intended to be used using a dictionary. Then, I'd add the comment below:
```
{{/*
Usage:
{{ include "elasticsearch.storageClass" (dict "global" .Values.global "local" .Values.master) }}
*/}}

Reply:
Alright, I looked at several other places (after messing it up once think I have it in the correct location per other items in the helper template.

Please let me know if that's not where it belongs!

Reply:
Perfect thanks!